assets/src/application/ui/input/PriceTypeSign/usePriceTypeHtmlClassName.tsx

```diff
@@ -0,0 +1,15 @@
+import { getCurrencySignCharacterCountClassName, getCurrencySignPositionClassName } from './utils';
```

Comment:
This file isn't currently used in `assets/src`, but since I've removed the old folder decided to update it a bit and add here for future usage(maybe).